src/smart-components/group/group.js

```diff
@@ -1,131 +1,57 @@
-  };
-
-  render() {
-    let { item } = this.props;
-
-    return (
-      <DataListItem key={ `group-${item.uuid}` }
-        aria-labelledby={ `check-group-${item.uuid}` }
-        isExpanded={ this.props.isExpanded(`group-${item.uuid}`) }>
-        <DataListToggle
-          onClick={ () => this.props.toggleExpand(`group-${item.uuid}`) }
-          isExpanded={ this.props.isExpanded(`group-${item.uuid}`) }
-          id={ `group-${item.uuid}` }
-          aria-labelledby={ `group-${item.uuid} group-${item.uuid}` }
-          aria-label="Toggle details for"
-        />
-        <DataListCheck aria-labelledby={ `check-group-${item.uuid}` } name={ `check-group-${item.uuid}` }/>
-        <DataListCell>
-          <span id={ item.uuid }>{ item.name } </span>
-        </DataListCell>
-        <DataListCell>
-          { this.fetchUserListForGroup(item) }
-        </DataListCell>
-        <DataListCell
-          class="pf-c-data-list__action"
-          aria-labelledby={ `group-${item.uuid} check-group-action${item.uuid}` }
-          id={ `group-${item.uuid}` }
-          aria-label="Actions">
-          { this.buildGroupActionKebab(item) }
-        </DataListCell>
-        <DataListContent aria-label="Group Content Details"
-          isHidden={ !this.props.isExpanded(`group-${item.uuid}`) }>
-          <Stack gutter="md">
-            <StackItem>
-              <Title size="md">Description</Title>
-            </StackItem>
-            <StackItem>
-              <TextContent component={ TextVariants.h6 }>
-                { item.description }
-              </TextContent>
-            </StackItem>
-            <StackItem>
-            </StackItem>
-            <StackItem>
-              <Title size="md">Members</Title>
-            </StackItem>
-            <StackItem>
-              <TextContent component={ TextVariants.h6 }>
-                { this.fetchUserListForGroup(item) }
-              </TextContent>
-            </StackItem>
-          </Stack>
-        </DataListContent>
-      </DataListItem>
-    );
-  };
-}
+import React, { Fragment, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Route, Switch, withRouter } from 'react-router-dom';
+import AppTabs from '../app-tabs/app-tabs';
+import { TopToolbar, TopToolbarTitle } from '../../presentational-components/shared/top-toolbar';
+import { fetchGroup } from '../../helpers/group/group-helper';
+import GroupPolicies from './policy/policies';
+import GroupPrincipals from './principal/principal';
+
+const Group = ({ match: { params: { uuid }}, location: { pathname }}) => {
+  const breadcrumbsList = () => [
+    { title: 'Groups', to: '/groups' },
+    { title: 'Group', isActive: true }
+  ];
+  const [ isFetching, setFetching ] = useState(true);
+  const [ group, setGroup ] = useState(true);
+  const [ tabItems ] = useState([{ eventKey: 0, title: 'Members', name: `${pathname}/members` },
```

Comment:
`tabItems` is never changed, there is no point of having it as state variable. Just use plain constant somewhere.